crates/forge/src/lib.rs

```diff
@@ -164,6 +169,21 @@ pub async fn run(
     Ok(summaries)
 }
 
+fn warn_if_available_gas_used_with_incompatible_scarb_version(
```

Comment:
I don't think it's a suitable place to have this function

Reply:
We already have the `warn_if..` function in the `main.rs` because it is used only there. I don't think there is a need to create separate file just for this function